### PR TITLE
Editorial: add reference for MP_NEW_CONN_ID frame

### LINKED ISSUE #404: Editorial: Refere to frame type
Section 1.2:
"Further, connection IDs are issued per Path ID." 

Should this refer to MP_NEW_CONNECTION_ID to be clear that there are new frames for this.

### PR DESCRIPTION
**Pull request by Yanmei-Liu**
resolves #404

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ The same Path ID is used in both directions to
 address a path in the new multipath control frames,
 such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
 PATH_AVAILABLE {{path-available-frame}} as well as MP_ACK {{mp-ack-frame}}.
-Further, connection IDs are issued per Path ID.
+Further, connection IDs are issued per Path ID (specified in {{mp-new-conn-id-frame}}).
```
Comment (gloinul) [inline draft-ietf-quic-multipath.md:205]: I think "using" rather "specified in" here would be more correct, or possibly "see" if one to be non-specific.

Comment (Yanmei-Liu) [reply draft-ietf-quic-multipath.md]: Indeed. I updated.

**Review by gloinul — APPROVED** [review]

**Review by huitema — APPROVED** [review]
I think this is OK. I am waiting for someone to argue that once the multipath extension is negotiated, even NEW_CONNECTION_ID frames do allocate connection ID per path -- but only for path = 0...

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -202,7 +202,7 @@ The same Path ID is used in both directions to
 address a path in the new multipath control frames,
 such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
 PATH_AVAILABLE {{path-available-frame}} as well as MP_ACK {{mp-ack-frame}}.
-Further, connection IDs are issued per Path ID.
+Further, connection IDs are issued per Path ID (using {{mp-new-conn-id-frame}}).
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:205]:
```suggestion
Further, connection IDs are issued per Path ID using the new
MP_NEW_CONNECTION_ID frame (see {{mp-new-conn-id-frame}}).
```

**Review by mirjak — CHANGES_REQUESTED** [review]
See proposal.

**Pull request merged**